content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
+
+[![wasm bevy examples](bevy_examples_wasm.png)](/examples)
+
+### Infinite Reverse Z Perspective Projection
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump)</div>
+
+For improved precision in the "useful range", the industry has largely adopted "reverse projections" with an "infinite" far plane. The new Bevy renderer was adapted to use the "right handed infinite reverse z" projection. [This Nvidia article](https://developer.nvidia.com/content/depth-precision-visualized) does a great job of explaining why this is so worthwhile. 
+
+### Compute Shaders
+
+The new renderer makes it possible for users to write compute shaders. Our new ["compute shader game of life" example](https://github.com/bevyengine/bevy/blob/main/examples/shader/compute_shader_game_of_life.rs) illustrates how to write compute shaders in Bevy.
+
+![compute game of life](compute.png)
+
+
+### New Multiple Windows Example
+
+<div class="release-feature-authors">authors: @DJMcNab</div>
+
+The "multiple windows" example has been updated to use the new renderer apis. Thanks to the new renderer apis, this example is now [much nicer to look at](https://github.com/bevyengine/bevy/blob/main/examples/window/multiple_windows.rs) (and will look even nicer when we add high level Render Targets).
+
+![multiple windows](multiple_windows.png)
+
+### Crevice
+
+<div class="release-feature-authors">authors: @cart, @mockersf</div>
+
+Bevy's old `Bytes` abstraction has been replaced with a fork of the [crevice](https://github.com/LPGhatguy/crevice) crate, which makes it possible to write normal Rust types to GPU-friendly data layouts. Namely std140 (uniform buffers default to this layout) and std430 (storage buffers default to this layout). Bevy exports `AsStd140` and `AsStd430` derives:
+
+```rust
+#[derive(AsStd140)]
+pub struct MeshUniform {
+    pub transform: Mat4,
+    pub inverse_transpose_model: Mat4,
+}
+```
+
+Coupling an `AsStd140` derive with our new `UniformVec<T>` type makes it easy to write Rust types to shader-ready uniform buffers:
+
+```rust
+// WGSL shader
+struct Mesh {
+    model: mat4x4<f32>;
+    inverse_transpose_model: mat4x4<f32>;
+};
+
+[[group(2), binding(0)]]
+var<uniform> mesh: Mesh;
+```
+
+### UV Sphere Mesh Shape
+
+<div class="release-feature-authors">authors: @nside</div>
+
+Bevy now has a built-in "uv sphere" mesh primitive.
+
+```rust
+Mesh::from(UVSphere {
+    radius: 1.0,
+    sectors: 16,
+    stacks: 32,
+})
+```
+
+![uv sphere](uv_sphere.png)
+
+### Flat Normal Computation
+
+<div class="release-feature-authors">authors: @jakobhellermann</div>
+
+The `Mesh` type now has a `compute_flat_normals()` function. Imported GLTF meshes without normals now automatically have flat normals computed, [in accordance with the GLTF spec](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#meshes). 
+
+![flat normals](flat_normals.png)
+
+### Faster GLTF Loading
```

Comment:
Maybe worth noting that support for loading compressed textures is in the works which also enormously speeds up loading non-trivial scenes?